### PR TITLE
Fixed broken relative links

### PR DESCRIPTION
**Pull request by n-boshnakov**
**What this PR does / why we need it**:
This PR fixes the broken relative links found in the external Gardener website.
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

**Comment by gardener-robot-ci-2** [comment]
Thank you @n-boshnakov for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

**Review by Kristian-ZH — APPROVED** [review]

File: docs/deployment/aks.md [hunk 1]
```diff
@@ -299,7 +299,7 @@ are.
 # Create a CloudProfile
 
 We need to create a CloudProfile to be referred from the Shoot
-([`example/30-cloudprofile-azure.yaml`](../../example/30-cloudprofile-azure.yaml)):
+([`example/30-cloudprofile-azure.yaml`](../../example/30-cloudprofile.yaml)):
```
Comment (Kristian-ZH) [inline docs/deployment/aks.md]:
```suggestion
([`example/30-cloudprofile.yaml`](../../example/30-cloudprofile.yaml)):
```

File: docs/deployment/aks.md [hunk 2]
```diff
@@ -328,7 +328,7 @@ about having a valid Kubeconfig to talk to its API.
 Lets start with the required seed secret first. Here we need to
 provide it's cloud provider credentials and kubeconfig in the seed
 secret. Update
-[`example/40-secret-seed-azure.yaml`](../../example/40-secret-seed-azure.yaml)
+[`example/40-secret-seed-azure.yaml`](../../example/40-secret-seed.yaml)
```
Comment (Kristian-ZH) [inline docs/deployment/aks.md]:
```suggestion
[`example/40-secret-seed.yaml`](../../example/40-secret-seed.yaml)
```

File: docs/deployment/aks.md [hunk 3]
```diff
@@ -357,7 +357,7 @@ kubectl apply -f example/40-secret-seed-azure.yaml
 ```
 
 Before creating Seed, we need to update the
-[`example/50-seed-azure.yaml`](../../example/50-seed-azure.yaml) file and
+[`example/50-seed-azure.yaml`](../../example/50-seed.yaml) file and
```
Comment (Kristian-ZH) [inline docs/deployment/aks.md]:
```suggestion
[`example/50-seed.yaml`](../../example/50-seed.yaml) file and
```

File: docs/deployment/aks.md [hunk 4]
```diff
@@ -454,15 +454,15 @@ projects:
 ## Create a SecretBinding and related Secret
 
 We'll use same Azure credentials with
-[`example/40-secret-seed-azure.yaml`](../../example/40-secret-seed-azure.yaml),
+[`example/40-secret-seed-azure.yaml`](../../example/40-secret-seed.yaml),
```
Comment (Kristian-ZH) [inline docs/deployment/aks.md]:
```suggestion
[`example/40-secret-seed.yaml`](../../example/40-secret-seed.yaml),
```

File: docs/deployment/aks.md [hunk 5]
```diff
@@ -454,15 +454,15 @@ projects:
...
 this is due to the fact that we use the same Azure Subscription for
 the Shoot and Seed clusters. Differently from the Seed secret, in this
 one we don't need to provide `kubeconfig` since the Shoot cluster will
 be provisioned by Gardener, and we need to provide credentials for
 Route53 DNS records management.
 
 Update
-[`example/70-secret-cloudprovider-azure.yaml`](../../example/70-secret-cloudprovider-azure.yaml)
+[`example/70-secret-cloudprovider-azure.yaml`](../../example/70-secret-cloudprovider.yaml)
```
Comment (Kristian-ZH) [inline docs/deployment/aks.md]:
```suggestion
[`example/70-secret-cloudprovider.yaml`](../../example/70-secret-cloudprovider.yaml)
```

File: docs/deployment/aks.md [hunk 6]
```diff
@@ -496,7 +496,7 @@ kubectl apply -f example/70-secret-cloudprovider-azure.yaml
 
 And create the SecretBinding resource to allow Gardener use that
 secret
-([`example/80-secretbinding-cloudprovider-azure.yaml`](../../example/80-secretbinding-cloudprovider-azure.yaml)):
+([`example/80-secretbinding-cloudprovider-azure.yaml`](../../example/80-secretbinding-cloudprovider.yaml)):
```
Comment (Kristian-ZH) [inline docs/deployment/aks.md]:
```suggestion
([`example/80-secretbinding-cloudprovider.yaml`](../../example/80-secretbinding-cloudprovider.yaml)):
```

File: docs/deployment/aks.md [hunk 7]
```diff
@@ -510,7 +510,7 @@ problems reported.
 
 ## Create the Shoot resource
 
-Update the fields in [`example/90-deprecated-shoot-azure.yaml`](../../example/90-deprecated-shoot-azure.yaml):
+Update the fields in [`example/90-deprecated-shoot-azure.yaml`](../../example/90-deprecated-shoot.yaml):
```
Comment (Kristian-ZH) [inline docs/deployment/aks.md]:
```suggestion
Update the fields in [`example/90-deprecated-shoot.yaml`](../../example/90-deprecated-shoot.yaml):
```

**Review by Kristian-ZH — CHANGES_REQUESTED** [review]

**Review by rfranzke — COMMENTED** [review]
Honestly, I think we can delete this `aks.md` document entirely. It is heavily outdated and obviously not in-use.

**Comment by n-boshnakov** [comment]
> Honestly, I think we can delete this `aks.md` document entirely. It is heavily outdated and obviously not in-use.

File was deleted in the [Deleted aks.md file](https://github.com/gardener/gardener/pull/4571/commits/de433c99ba71e0c2f16d7144967bdc50b39b71fc) commit.

**Review by rfranzke — CHANGES_REQUESTED** [review]
Please also delete it from here:
https://github.com/gardener/gardener/blob/master/docs/README.md#deployment

/squash

**Comment by n-boshnakov** [comment]
> Please also delete it from here:
> https://github.com/gardener/gardener/blob/master/docs/README.md#deployment
> 
> /squash

Removed reference to AKS in [Removed reference to deleted file](https://github.com/gardener/gardener/pull/4571/commits/e35e6c8c085f728d71da381f237df49a8e8d1274).

**Review by rfranzke — APPROVED** [review]
/lgtm
/reviewed ok-to-test

**Pull request merged**